.github/stale.yaml

```diff
@@ -0,0 +1,18 @@
+# Number of days of inactivity before an issue becomes stale
+daysUntilStale: 60
+# Number of days of inactivity before a stale issue is closed
+daysUntilClose: 7
+# Issues with these labels will never be considered stale
+exemptLabels:
+  - pinned
+  - security
```

Comment:
We do not have these labels?

Reply:
creating it :)

Reply:
@nixpanic its available :)

Reply:
Some description in the developers document about the labels would be nice. But at minimum there needs to be a text added to the labels in GitHub. Nobody would know what an undocumented `pinned` label means.

Reply:
ok.. kind of realligned or following most of the settings as in rook repo. added some description to the labels too, may be it can be enhanced later.